mubench.reviewsite/src/Model/Misuse.php

```diff
@@ -136,6 +136,12 @@ public function hasSufficientReviews()
         return $this->getReviewState() > ReviewState::NEEDS_REVIEW;
     }
 
+    public function isDistinctReviewState()
```

Comment:
I couldn't deduce what this function does from its name. I thought about renaming it to `hasConclusiveReviewState()`, but that's not it either, because its `true` on `NEEDS_REVIEW`. In lack of a good name that captures this intention, I would probably change this function to also return `false` on `NEEDS_REVIEW` and then additionally check for `!$misuse->hasSufficientReviews()` in the filtering. Or maybe you have a better suggestion?